source/dr-ha/draas/veeam_draas/configuring_a_replication_job.md

```diff
@@ -0,0 +1,94 @@
+
+![Create Replication Job](files/createreplicationjob/createreplicationjob.png)
+
+### Name
+1. Enter a name for the replication job and a description if needed (something that makes it easy to reference which VMs are in the job)
+    1. **Replica seeding** - Used if there has been a copy of your backups shipped to us and uploaded on to our Veeam environment.
+    2. **Network remapping** - <span style="color:green">**(enable)**</span> Used to map the internal networks on your current VMs to the internal networks UKFast has provided to you.
+    3. **Replica re-IP** - <span style="color:red">**(do not enable)**</span> - Used when your DR sites use different internal networks to your production. This is not required when replicating to UKFast, as you will keep your current internal IPs.
+
+![Replication Job - Name](files/createreplicationjob/createreplicationjob_name.png)
+
+### Virtual Machines
+1. **Add** - Select the VMs that you would like to replicate in this job<br />
+    1. **Exclusions** - You have the option to exclude specific disks on VMs from being replicated. You may want to do this if you want to save your Cloud Connect storage and there is a non-critical data disk that won't be needed in the event of a DR situation.
+    2. **Source** - You can choose to obtain the replication data from a Veeam backup rather than taking it directly from a snapshot on the VM. This is useful if you want to limit the snapshots on a VM and there aren't strict RPO requirements. <span style="color:red">**This will only replicate data from the latest backup, so enable with caution.**</span>
+
+![Replication Job - Virtual Machine](files/createreplicationjob/createreplicationjob_virtualmachine.png)
+
+
+### Destination
+1. Under "Host or cluster" select **Choose** > **Cloud host** > select the UKFast Service Provider (UKFast’s external DNS name with your login name after in brackets) > Click "OK"
+2. Under "Datastore" ensure that your datastore is select. If you have multiple, select the relevant one.
+
+![Replication Job - Destination](files/createreplicationjob/createreplicationjob_destination.png)
+
+### Network
+
+Here you’ll create your network mappings. When you create a mapping all VMs on the "source network" at your site will be connected to the "target network" at UKFast. You should have a dedicated target network for each of the source networks which you are replicating VMs from.
+
+
+1. Select "Add" to create a network mapping > select the "source network" and "target network"
+
+![Replication Job - Network](files/createreplicationjob/createreplicationjob_network.png)
+
+### Job Settings
+1. **Repository for replica metadata** - Select a local repository for replica metadata. You will need on average 128MB of storage for every 1TB replicated, but this could vary.
+2. **Replica name suffix** - This shows up within UKFast’s VMware environment, so we recommend that it is left as the default "_replica"
```

Comment:
Unsure if the unescaped underscore will cause problems with formatting. Will have to check.